### ISSUE TITLE
Keep an eye on calamares installer

### ISSUE THREAD

**Issue by blshkv** [pentoo/pentoo-livecd#7]
```Calamares is an installer framework. By design it is very customizable, in order to satisfy a wide variety of needs and use cases```

Our previous installer was badly written. @Wuodan patched it quite a bit but it is still broken. So in short, we don't have a well supported and a proper installer. It's a hack written on shell.

@necrose99 suggested (https://github.com/pentoo/pentoo-installer/issues/9) https://calamares.io/about/
and it might be a better choice indeed. I suggest to keep an eye on it.

According to a Gentoo dev ( https://blogs.gentoo.org/johu/ ) it will be ready for production soon and Sabayon might use it for a next release.

**Comment by ZeroChaos-** [comment]
yes, one broken installer or another right?  none of this matters except the fact that I can fix ours to work better but calamares doesn't work at all yet.

**Comment by necrose99** [comment]
for now its merely tinkering , so far for sabayon its working but yet system-d 
but compared to how good Anaconda was then went to shit , basically the installer became working on one iso and not others erratic with new UI change.  old ui worked perfectly.... 
but it spins up slow , but it works other than Cypto. /openrc issues since like 15+++ distros use... hopefully it matures quickly. 
https://packages.gentoo.org/packages/app-crypt/zuluCrypt  is an after the fact noob-ish work about. 

I'm merely mocking up the slide **art work** for now as calamares is still not quite their 
oven time is not quite even baked.  but as **Gentoo works** with it *perhaps it will become more useful*. 

**A: keep the "expert mode" installer** for those with a clue and not the **sql T-shirt on Think geek**... 
**B: GUI for the lazy or noobs . perhaps much latter** new to Pentoo or students .... it wont be anytime soon....  (ugghh goto break them in somehow /groan) and **better to raise the bar** instead of something using apt......ish..... 

Anyhow I just hopefully can make a few nice slides at the least , and hopeful it will keep 'someone' else of the necks for a min, for now i'll take art critiques on the slides ,times a plenty ... meh re-purpose them for something ..... 

i'd rather handed over in a more working state. i was going to keep quiet till more working than purely theoretical construct... 
 but hopefully as our **frequent curious guest** salivates over it, will dig more on it. and or fork it and toy with it.  its perfectly lab grade... 
*buy a moments respite*...... 

**CCNA , Security+ CCNA security , CISSP , resume Masters** , got me bleeping hands full up.  
Certs I've kinda skinched on them but the HR-DEPT-Firewall of brainwashed..... no cert no job of late so I bow... 

**if our curious guest wants to take it on then i say let him... ** better him than me ... 

however till after FEB/march my hands are greatly tied up. 
more or less trying to get the dockerhub to eat the 2.5 gig tar.bz2 of pentoo and build a working docker image... so far the dockerhub tends to choke on SF.net load (cloud drive service would kill my account in time if kept Pentoo tarballs their.)
or i may be forced to build them and push as a raw squash-base container ,call into a pentoo-arch$-builder  to build anything new fresh. **may also add some ease to the binhost.**  
but so far i have a good stage 4 packager , adding a get me a pentoo-$ARCH-$subarch.iso >  /pentoo-$ARCH-$subarch-stage5.tar.bz2  grep ??? iso  only thing i dont have on it  ie look for iso change name get iso name wget iso , 

Likewise also like to get vagrant to stamp out Pentoo Virtualbox'es and vmware and LXC ,vagrant etc. 
**perhaps it will ease a certain nagging ** 
...  iso Symlinks have aided so far thank you. 
pentoo  installer I've carved it up , into smaller scripts , as to affect the install of pentoo into virtualbox /etc in a automated way , just so far the **json niceties of syntax** keep boning me.  its not at all forgiving.  pull into /root/kernel-configs/pentoo-defaco-kernel.conf (spares for the new-ish) 
 https://atlas.hashicorp.com/session is capable of pulling pentoo into a vbox /etc for us. as well its a quick and expendable testing platform when required and a tool to teach newer users without going .....ape.... 

calamares   has much time to grow , however it should keep our** curious guest** full of "Edutainment" to try n F with it.  I see getting a docker set of containers and vagrant/packer more achievable.. so far its not letting my dyslexia bend me over that crappy json syntax and debugging the bash to get that to build boxes reliably. 
finding the bleeping time.... too...

**Comment by blshkv** [comment]
https://calamares.io/calamares-2.3-is-out/
It's getting better.

**Comment by blshkv** [comment]
The big focus of the Calamares 2.3 development cycle was implementing support for installing with full disk encryption. 

Other highlights of this release include:
fixed a long standing and hard to debug issue with occasional unwanted resize operations being executed;
added a partitioning module configuration option for showing partition labels in the manual partitioning interface;
improved support for NVME devices in fstab;
much improved mke2fs and e2fsck handling, which fixes a multitude of random (but not very frequent) install failures;
improved restart command configuration handling in the finished module;
implemented type to search support in the keyboard module;
added a partitioning module configuration option for selecting the default filesystem type;
countless other bug fixes.

**Comment by necrose99** [comment]
https://github.com/necrose99/calamares-pentoo-branding
i swiped some stuff from sabayon , mainly branding templates artworks not all done but for test demo purposes it works , im sure you all have better black-hat etc type of pics etc that better feature a Pentoo  release. however for demo purposes for a lab spin i did my best. 

the text installer is'nt all bad but have a gui -installer for newb's  might be of use.  with work and travel , grad classes Dallas/Houston I haven't had as much time of late to work on stuff....

**Comment by blshkv** [comment]
https://www.sabayon.org/article/sabayon-linux-development-2015

**Comment by mudler** [comment]
you may want to look at https://github.com/calamares/calamares-manjaro for the openrc integration
